plugins/permission-backend-module-policy-allow-all/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@backstage/plugin-permission-backend-module-policy-allow-all",
+  "description": "Allow all policy backend module for the permission plugin.",
+  "version": "0.0.0",
```

Comment:
Is this the correct version for the first release of a package?

Reply:
Yes, along with a `minor` changeset